change_color.sh

```diff
@@ -158,7 +164,14 @@ TERMINAL_COLOR11=${TERMINAL_COLOR11:-ef6c00}
 TERMINAL_COLOR12=${TERMINAL_COLOR12:-03a9f4}
 
 OUTPUT_THEME_NAME="${OUTPUT_THEME_NAME-oomox-$THEME}"
-DEST_PATH="$HOME/.themes/${OUTPUT_THEME_NAME/\//-}"
+DEST_PATH="${DEST_PATH-$HOME/.themes}"
+if [[ -d "$DEST_PATH" ]] ; then
+    DEST_PATH="$DEST_PATH/${OUTPUT_THEME_NAME/\//-}"
+    echo "building $OUTPUT_THEME_NAME in $DEST_PATH"
+else
+	DEST_PATH="$HOME/.themes/${OUTPUT_THEME_NAME/\//-}"
+    echo "can't find chosen target directory, building $OUTPUT_THEME_NAME in default target directory $DEST_PATH"
```

Comment:
this check doesn't really makes sense because of: https://github.com/actionless/oomox-gtk-theme/pull/26/files#diff-77c332e84f1c4beca18bbe3d1be580a3R180

Reply:
and also please use different variables for DEST_PATH_ROOT and DEST_PATH instead of reusing the same variable

Reply:
Thanks. I hadn't considered that the mkdir would mop up any problems with the input there. Amended, tested and I can't break it. I've also tidied up the formatting on the help section entry and amended the input variable to DEST_PATH_ROOT.